javascript/ql/lib/qlpack.yml

```diff
@@ -1,7 +1,8 @@
 name: codeql/javascript-all
-version: 0.0.3
+version: 0.0.4
```

Comment:
Do we want to bump all of our libraries to 0.0.4 so we can at least start with consistency?

Reply:
I thought about that, but all our version numbers will diverge soon enough anyway, so I figured I'd leave the inconsistency in there so people get used to it:)